### PR TITLE
Modified 2.-workspaces/m.groups.md

### PR DESCRIPTION
**Pull request by sydstoplight**

### CODE REVIEW

File: 2.-workspaces/m.groups.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Groups allow you to organize projects in your sidebar and grant project access t
 
 https://www.youtube.com/watch?v=d5e5Ac-6_U4
 
+1. Log in to your workspace, and then select the **New Group** icon at the right of the header pane.
```
Comment (pamgoodrich) [inline 2.-workspaces/m.groups.md:12]: I think you need to select the Groups tab first, and then select New Group.

File: 2.-workspaces/m.groups.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ Groups allow you to organize projects in your sidebar and grant project access t
...
+2. From the **Create New Group** page, provide a name for your group and set the [project visibility](https://meta.stoplight.io/docs/platform/ZG9jOjg1NjcyNzE-manage-project-accessundefined).
```
Comment (pamgoodrich) [inline 2.-workspaces/m.groups.md:13]: I would use the short link for this topic. The best way to do this is to find the file you want to link to in the Files tab, right click, and then select Copy Path. If the file is in a different folder than the one you are updating, you need to select Copy Relative Path.

File: 2.-workspaces/m.groups.md [hunk 3]
```diff
@@ -27,10 +30,16 @@ There are a couple ways to move projects into a group.
 
 ![project_list_move_project.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/za5fPvBFhDg)
 
+1. Log in to your workspace, and then select the **ellipsis** icon (...) at the right of the project you wish to move.
```
Comment (pamgoodrich) [inline 2.-workspaces/m.groups.md:33]: You need to select the Projects tab first.

File: 2.-workspaces/m.groups.md [hunk 4]
```diff
@@ -27,10 +30,16 @@ There are a couple ways to move projects into a group.
...
 ![project_list_move_project.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/za5fPvBFhDg)
 
+1. Log in to your workspace, and then select the **ellipsis** icon (...) at the right of the project you wish to move. 
+2. Select **Move Project**.
+
 **Option 2: Move Multiple Projects to a Group from the Projects List**
 
 ![project_settings.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/0Lt57dlIpDc)
 
+1. Log in to your workspace, and then select the **checkbox** icon at the left of the **Name** filter.
```
Comment (pamgoodrich) [inline 2.-workspaces/m.groups.md:40]: Same here. You need to select the Projects tab first.

File: 2.-workspaces/m.groups.md [hunk 5]
```diff
@@ -27,10 +30,16 @@ There are a couple ways to move projects into a group.
...
+2. From the **Move** page, select a group from the dropdown to move selected projects.
```
Comment (pamgoodrich) [inline 2.-workspaces/m.groups.md:41]: change dropdown to menu

**Comment by pamgoodrich** [comment]
@sydstoplight : I requested a few changes. After you make these changes and push them, I will merge.

Thank you!

File: 2.-workspaces/m.groups.md (hunk 1 above)
Comment (sydstoplight) [reply 2.-workspaces/m.groups.md:12]: Added step

File: 2.-workspaces/m.groups.md (hunk 2 above)
Comment (sydstoplight) [reply 2.-workspaces/m.groups.md:13]: Changed to short link

File: 2.-workspaces/m.groups.md (hunk 3 above)
Comment (sydstoplight) [reply 2.-workspaces/m.groups.md:33]: Added step

File: 2.-workspaces/m.groups.md (hunk 4 above)
Comment (sydstoplight) [reply 2.-workspaces/m.groups.md:40]: Added step

File: 2.-workspaces/m.groups.md (hunk 5 above)
Comment (sydstoplight) [reply 2.-workspaces/m.groups.md:41]: Changed wording

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**